idb/idb.go

```diff
@@ -332,12 +338,27 @@ type AssetBalanceRow struct {
 	Deleted      *bool
 }
 
-// ApplicationRow is metadata relating to one application in an application query.
+// ApplicationRow is metadata and global state (AppParams) relating to one application in an application query.
 type ApplicationRow struct {
 	Application models.Application
 	Error       error
 }
 
+// ApplicationQuery is a parameter object used for query local and global application state.
+type ApplicationQuery struct {
+	Address                  []byte
```

Comment:
this should be `basics.Address`

Reply:
I had assumed it was better to keep the existing pattern in idb.go for []byte addresses than introduce an inconsistency, wdyt?

Reply:
I know this will make an inconsistency, but I think it's ok as long as the code is moving in the right direction.